package.json

```diff
@@ -45,7 +45,9 @@
     "babel-polyfill": "6.26.0",
     "babel-register": "6.26.0",
     "bootstrap": "4.3.1",
+    "classnames": "^2.2.6",
     "date-fns": "1.30.1",
+    "jquery": "^1.9.1",
```

Comment:
How are these changes necessary for deployment? Revert if not.

Reply:
This fix a complaint about missing jquery, from bootstrap. it seems to work without it anyway but keeps complaining.

Reply:
Please file it as a separate PR. Anyone on the team can take on these kinds of minor updates that are unrelated to the task of auto-deployment.